apps/src/levelbuilder/skills/SkillsApi.tsx

```diff
@@ -0,0 +1,76 @@
+import MetricsReporter from '@cdo/apps/metrics/MetricsReporter';
+import {MetricEvent} from '@cdo/apps/metrics/events';
+import {getAuthenticityToken} from '@cdo/apps/util/AuthenticityTokenStore';
+
+interface Skill {
+  key: string;
+  description: string;
+  evaluationCriteria: string;
+  concept: string;
+}
+
+export async function createSkill(skill: Skill) {
+  try {
+    const response = await fetch(`/skills`, {
```

Comment:
I've started using `HttpClient.fetchJson` ([here](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/util/HttpClient.ts#L55)) because it is much easier to mock in tests and it automatically adds the auth token. Completely fine to keep using the global fetch, just thought I'd mention in case you hadn't seen it before.

Reply:
I went down this [Slack rabbit hole](https://codedotorg.slack.com/archives/C0T0PNR0D/p1729198686870329) and am all for standardizing. 🙃 Will update. Thanks for bringing this up!